### PR TITLE
Disable datepicker for disabled or readonly fields

### PR DESCRIPTION
**Pull request by flamerohr**
The fields were still editable through the datepicker, which could have undesirable effects.

### CODE REVIEW

File: admin/client/dist/js/LeftAndMain.js [hunk 1]
```diff
@@ -289,7 +289,7 @@
         data.__forceReferer = forceReferer;
 
         if (forceReload) {
-          data.__forceReload = Math.random();
+          data.__forceReload = 1 + Math.random();
```
Comment (dhensby) [inline admin/client/dist/js/LeftAndMain.js]: This is unrelated and part of another PR (I believe)?

**Comment by dhensby** [comment]
Seems legit - need the sanity checks to pass first

File: admin/client/dist/js/LeftAndMain.js (hunk 1 above)
Comment (flamerohr) [inline admin/client/dist/js/LeftAndMain.js]: yeah, that was related to the hash fix @tractorcow did on Friday
